lib/utils/object.ts

```diff
@@ -46,8 +46,10 @@ export function deepAssign(target: any, ...sources: any[]): any {
         } else if (sourceValue instanceof Date) {
 
           targetValue = new Date(sourceValue);
-        } else {
+        } else if (sourceValue === null) {
```

Comment:
above L39 condition of
`} else if (typeof sourceValue === 'object') {`
is true if sourceValue = null.

![image](https://user-images.githubusercontent.com/1043276/29368741-90ef0754-82db-11e7-83d1-46949b6c271b.png)